packages/dx-react-chart/src/types/templates.axis.types.ts

```diff
@@ -0,0 +1,37 @@
+export interface LabelProps {
+  // The label text
+  text: string | number;
+  // The x coordinate of the label’s top left corner
+  x: number;
+  // The y coordinate of the label’s top left corner
+  y: number;
+  // The label’s baseline
+  dominantBaseline: string;
+  // The label’s text alignment
+  textAnchor: string;
+}
+export interface LineProps {
+  // The line start’s x coordinate
+  x1: number;
+  // The line end’s x coordinate
+  x2: number;
+  // The line start’s y coordinate
+  y1: number;
+  // The line end’s y coordinate
+  y2: number;
+}
+
+export interface RootProps {
+  // The x coordinate of the top left corner of the axis’ rendering area
+  dx: number;
+  // The y coordinate of the top left corner of the series’ rendering area
+  dy: number;
+  /** @internal */
+  onSizeChange: any;
```

Comment:
Why `any`? It is declared in *dx-react-core* - `SizerProps.onSizeChange`.

Reply:
I'll fix it